platform/src/core/apis/Account.js

```diff
@@ -47,34 +42,34 @@ export const updateUserDetails = async (userID, data) => {
 };
 
 export const getAssignedGroupMembers = async (groupID) => {
-  return await axios
+  return await createAxiosInstance(false)
```

Comment:
Hi @OchiengPaul442 , please shed more light on this `false`? Why is it a` false`?

Reply:
@Baalmart, Error on my part was testing with the token because of a 401 error on my side and forgot to remove the fales.

Reply:
Cool, thanks @OchiengPaul442